mod_ngarn/utils.py

```diff
@@ -45,3 +53,35 @@ class ModuleNotfoundException(Exception):
             return globals()["__builtins__"][fn_name]
     except KeyError as e:
         raise ImportNotFoundException(e)
+
+
+async def init_table(table: str = escape_table_name(os.getenv('DBTABLE', 'modngarn_job'))):
+    cnx = await get_connection()
+    async with cnx.transaction():
+        await cnx.execute(
+            """CREATE TABLE IF NOT EXISTS {table} (
+                    id TEXT NOT NULL CHECK (id !~ '\\|/|\u2044|\u2215|\u29f5|\u29f8|\u29f9|\ufe68|\uff0f|\uff3c'),
+                    fn_name TEXT NOT NULL,
+                    args JSONB DEFAULT '[]',
+                    kwargs JSONB DEFAULT '{}',
```

Comment:
Use this `kwargs JSONB DEFAULT '{{}}'`

Reply:
Why `JSONB` and not `JSON` (or even `text`) for these columns?